lib/python/qmk/cli/generate/autocorrect_data.py

```diff
@@ -63,7 +63,18 @@ def parse_file(file_name: str) -> List[Tuple[str, str]]:
   """
 
     try:
-        from english_words import english_words_lower_alpha_set as correct_words
+        import english_words
+        correct_words = english_words.get_english_words_set(['web2'], lower=True, alpha=True)
+    except AttributeError:
+        try:
+            from english_words import english_words_lower_alpha_set as correct_words
+        except:
```

Comment:
CI doesn't like the bare except here.

Reply:
Sorry, first time contributing here. I didn't run the tests locally, I expected the tests to run on the PR creation. I'll get that sorted. I was trying to provide a message there rather than bomb the process, but really, it's such a low probability code path it can probably be removed.